src/SFA.DAS.ProviderCommitments/SFA.DAS.ProviderCommitments.Web/Controllers/DraftApprenticeshipController.cs

```diff
@@ -110,7 +116,53 @@ public async Task<IActionResult> AddDraftApprenticeship(AddDraftApprenticeshipVi
             }
         }
 
-        private async Task AddLegalEntityAndCoursesToModel(AddDraftApprenticeshipViewModel model)
+        [HttpGet]
+        [Route("edit-apprentice/{DraftApprenticeshipHashedId}")]
+        public async Task<IActionResult> EditDraftApprenticeship(EditDraftApprenticeshipRequest request)
+        {
+            if (!ModelState.IsValid)
+            {
+                return BadRequest(ModelState);
+            }
+
+            var imodel = await _providerCommitmentsService.GetDraftApprenticeshipForCohort(request.CohortId.Value, request.DraftApprenticeshipId.Value);
+            var model = _editDraftApprenticeshipDetailsToViewModelMapper.Map(imodel);
+
+            await AddLegalEntityAndCoursesToModel(model);
+
+            return View(model);
+        }
+
+        [HttpPost]
+        [Route("edit-apprentice/{DraftApprenticeshipHashedId}")]
```

Comment:
Routes. I think I prefer: 
https://localhost:5001/10005077/unapproved/JRML7V/apprentices/JRML7V/edit

To put this route in context, here are some more (touching on other stories, I know):
```
/10005077/unapproved/JRML7V/    				<--- cohort details page (GET)
/10005077/unapproved/JRML7V/apprentices/JRML7V			<--- draft apprentice details page (does not exist currently, maybe in enhanced approval it will) (GET)
/10005077/unapproved/JRML7V/apprentices/add			<--- add a new draft apprentice page (GET/POST)
/10005077/unapproved/JRML7V/apprentices/JRML7V/edit 		<--- edit an existing draft apprentice page (GET/POST)
```


Food for thought? 🥘

Reply:
We all agree this looks good